cilium-cli/connectivity/tests/world.go

```diff
@@ -96,7 +97,7 @@ func (s *podToWorld2) Run(ctx context.Context, t *check.Test) {
 
 	for _, client := range ct.ClientPods() {
```

Comment:
Not sure why we are managing the index outside the loop? Seems we could use for i, client := range t.Context().ClientPods{}. This also eliminate the need to declare ct var since it is only used in the scope of the loop?

Reply:
Discovered from a failing lint check: `ct.ClientPods()` returns `map[string]Pod`, so the key is a string and `Sprintf()` is expecting an int.

If it's okay I'm going to keep this as-is for now. Could revisit in a follow-up PR to clean this up everywhere (there's at least one other place in the file using the same `var i int` pattern)